spring-cloud-dataflow-server-core/pom.xml

```diff
@@ -183,11 +178,6 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-task-batch</artifactId>
 		</dependency>
-		<dependency>
```

Comment:
I see we still use it, are we picking this up transitively from somewhere else?

Reply:
`spring-boot-starter-test`